qualtran/surface_code/ccz2t_cost_model_test.py

```diff
@@ -30,3 +34,16 @@ def test_vs_spreadsheet():
     np.testing.assert_allclose(re.failure_prob, 0.0084, rtol=1e-3)
     np.testing.assert_allclose(re.footprint, 4.00e5, rtol=1e-3)
     np.testing.assert_allclose(re.duration_hr, 7.53, rtol=1e-3)
+
+
+def test_grid_search_runs():
+    # TODO: a better test vs some published reference
```

Comment:
I can add the MultiFactory and attempt to test this against THC. In my previous tries, the numbers are close but not quite match the paper exactly. I'm not sure about other published sources to check against.